docs/FONTS.md

```diff
@@ -0,0 +1,34 @@
+## Fonts API
+
+  - [Get Template Available Fonts](#get-template-available-fonts)
+
+### Get template available fonts
+
+Retrieves template available fonts.
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.getTemplateAvailableFonts(1021)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+
+[See example](/samples/fonts/get-template-available-fonts.js)
+
+`getTemplateVailabeFonts` function returns object containing all fonts. 
+The object which returned has method `getFontById` which accepts `fontId` as parameter.
+`getFontId` method return flatten font.
+
```

Comment:
Please write this way:
`getTemplateAvailabeFonts ` function returns an object containing all fonts. The returned object has `getFontById ` method which accepts `fontId` parameter. 
`getFontId` method returns flattened font.

Reply:
it is not understandable it return an id of single font or all ids of available fonts.

Reply:
Let's))